src/views/AddSubmission.js

```diff
@@ -19,6 +19,9 @@ import { Button } from 'react-bootstrap'
 import ResultsTable from '../components/ResultsTable'
 import ResultsAddModal from '../components/ResultsAddModal'
 import SubmissionRefsDeleteModal from '../components/SubmissionRefsDeleteModal'
+import TooltipTrigger from '../components/TooltipTrigger'
+//import Tooltip from "@material-ui/core/Tooltip"
```

Comment:
Please remove the commented-out dead `import` statement code, but this otherwise good-to-go. Approved, contingent on that change,